src/Gui/Clipping.cpp

```diff
@@ -42,8 +42,9 @@ using namespace Gui::Dialog;
 class Clipping::Private
 {
 public:
-    Ui_Clipping ui;
-    QPointer<Gui::View3DInventor> view;
+    static Ui_Clipping ui;
```

Comment:
Is this a good idea? It seems fishy to me to keep UI controls as static variables...

Reply:
My goal was to modify as little as possible to add this support, being unfamiliar with the code and libraries.  Making this static allowed me to essentially retain its existing lifecycle behavior and have to hook up fewer things.

It's maybe not a good idea, as your other related comments tell.